src/main/java/com/aws/greengrass/mqttclient/spool/Spool.java

```diff
@@ -300,10 +304,6 @@ public void persistentQueueSync(Iterable<Long> diskQueueOfIds, CloudMessageSpool
         int numMessages = 0;
         int queueOfMessageIdInitSize = queueOfMessageId.size();
         for (long currentId : diskQueueOfIds) {
-            // Check if Queue of message IDs already contains this ID
-            if (queueOfMessageId.contains(currentId)) {
```

Comment:
why is this removed?

Reply:
I have it in description.

Reply:
What happens if it ID does exist in the queue and you're wrong? Is that a problem?